webapp-django/crashstats/crashstats/views.py

```diff
@@ -2019,7 +2019,8 @@ def format_float(number):
             'product': r['product_name'],
             'version': r['version_string'],
             'percentage': format_float(r['percentage']),
-            'numberOfCrashes': r['report_count']})
+            'numberOfCrashes': r['report_count'],
+            'rank': r.get('currentRank', '')})
```

Comment:
The query that gets the data to here is this one: https://github.com/mozilla/socorro/blob/master/socorro/external/postgresql/signature_summary.py#L152-L179
No `currentRank`.